honestcash-v2/src/app/modules/editor/components/paid-section-selection/paid-section-selection.component.ts

```diff
@@ -23,7 +23,8 @@ export class EditorPaidSectionSelectionComponent implements OnInit, OnDestroy {
   @Input() form: NgForm;
   @ViewChildren('paidSectionElements') paidSectionElements: QueryList<ElementRef>;
   public LINEBREAK_ACTION = LINEBREAK_ACTION;
-  public isLoaded: boolean;
+  public saveStatus: EDITOR_STATUS;
+  public EDITOR_SAVE_STATUS = EDITOR_STATUS;
```

Comment:
Hmm, just reviewed this one in other PRs. These PRs may contain the same changes. Let's try to separate it in the future.